tests/networkpolicy_test.go

```diff
@@ -39,6 +41,16 @@ func assertPingFail(ip string, vmi *v1.VirtualMachineInstance) {
 	Expect(err).ToNot(HaveOccurred())
 }
 
+func waitForNetworkPolicyDeletion(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance, policyName string) {
+	Eventually(func() bool {
+		_, err := virtClient.NetworkingV1().NetworkPolicies(vmi.Namespace).Get(policyName, v13.GetOptions{})
+		if err != nil && !apierrors.IsNotFound(err) {
+			Fail(fmt.Sprintf("Unexpected error waiting for policy deletion: %v", err))
+		}
+		return apierrors.IsNotFound(err)
+	}, 10*time.Second, time.Second).Should(BeTrue(), fmt.Sprintf("Policy %s was not deleted", policyName))
```

Comment:
Please see [here](https://github.com/kubevirt/kubevirt/blob/master/tests/network/primary_pod_network.go#L123) on how to assert this and avoid this extra Should text.

Reply:
Interesting...

Reply:
```
		if err != nil && !errors.IsNotFound(err) {
			Fail(fmt.Sprintf("Unexpected error waiting for policy deletion: %v", err))
		}
```
Removing this then as well